### PR TITLE
Simply 449/add useful message

### PR DESCRIPTION
**Pull request by oliviawongnyc**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,37 +1,62 @@
 ## CHANGELOG
 
+### 07/23/21
+
+#### Updated
+
+- Added a message to be displayed if JS/CSS files cannot be found.
```
Comment (EdwinGuzman) [inline CHANGELOG.md]: "...cannot be found when the app is running and the`/admin` page is visited."

File: README.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ git clone https://github.com/NYPL-Simplified/registry_admin.git
 
 ## Installation (Docker)
 
+If not using Docker, skip to section entitled "Installation (non-Docker)."
```
Comment (EdwinGuzman) [inline README.md]: Not necessary but you can link to the heading by doing `["Installation (non-Docker)"](#installation-non-docker).`.

File: README.md [hunk 3]
```diff
@@ -124,4 +126,12 @@ To start the registry inside the virtualenv that `pipenv` creates:
...
 
-Pipenv should read in the local `.env` file and supply those database connection strings to the application, which will be run by the Flask development server.
\ No newline at end of file
+Pipenv should read in the local `.env` file and supply those database connection strings to the application, which will be run by the Flask development server.
+
+You can then navigate to http://localhost:5000/admin in the browser.
+
+### Debugging
+
+If you are served an error message on the admin home screen, you may need to run `npm install` in the root directory of the library_registry repo. If you're running the application locally, you can also try running the same command in the root directory of the registry_admin repo.
```
Comment (EdwinGuzman) [inline README.md]: You should add that these steps are for when you are running the app locally without Docker. For the first sentence, for production instances it's most likely the case that the app is running with Docker, so maybe you can add something that says to also look at the Docker configuration if that's what internally runs `npm install`.

File: README.md [hunk 4]
```diff
@@ -124,4 +126,12 @@ To start the registry inside the virtualenv that `pipenv` creates:
...
+
+The latter command will only work if the circulation-web and circulation repos are linked using `npm link`. To do this, run `npm link` in the registry_admin repo and then `npm link simplified-registry-admin` in the library_registry repo.
```
Comment (EdwinGuzman) [inline README.md:137]: Add that this is in the non-Docker setup.

File: CHANGELOG.md (hunk 1 above)
Comment (oliviawongnyc) [reply CHANGELOG.md]: Fixed!

File: README.md (hunk 2 above)
Comment (oliviawongnyc) [reply README.md]: Cool! Fixed!

File: README.md (hunk 3 above)
Comment (oliviawongnyc) [reply README.md]: Fixed!

File: README.md (hunk 4 above)
Comment (oliviawongnyc) [reply README.md:137]: Fixed!

**Review by EdwinGuzman — APPROVED** [review]

**Pull request merged**